tap_googleads/streams.py

```diff
@@ -100,24 +121,23 @@ def get_records(self, context: Optional[dict]) -> Iterable[Dict[str, Any]]:
         Yields:
             One item per (possibly processed) record in the API.
         """
-        if self.config.get("comma_separated_string_of_customer_ids", False):
-            customer_ids_list = self.config.get("comma_separated_string_of_customer_ids").replace(" ", "").split(",")
-            for i in customer_ids_list:
-                yield {"customerClient": {"id": str(i)}}
-        else:
-            for row in self.request_records(context):
-                row = self.post_process(row, context)
-                # Don't search Manager accounts as we can't query them for everything
-                if row["customerClient"]["manager"] == True:
-                    continue
-                yield row
+        for row in self.request_records(context):
```

Comment:
I think this might be because you have a google ads manager account that is not fully setup yet. But yeah good idea to catch this

Reply:
How would you propose we do this. I think `validate_response` returns either retriable or fatal error and in this case it should just ignore this manager account

Reply:
> I think this might be because you have a google ads manager account that is not fully setup yet. But yeah good idea to catch this

So we have two customers for which child contexts are generated from the `AccessibleCustomers` stream: one works fine (same ID as what is set for `login_customer_id`) and the other hits this `403` error. I'm not super familiar with Google Ads, so does that sound like a valid scenario that we should be handling (i.e. skip the customer) to you?

Reply:
> How would you propose we do this. I think `validate_response` returns either retriable or fatal error and in this case it should just ignore this manager account

```py
from http import HTTPStatus
```

```py
    def validate_response(self, response):
        if response.status_code == HTTPStatus.FORBIDDEN:
            self.logger.warning(...)   # something helpful
            return

        return super().validate_response(response)
```

Reply:
I think something like this could work but we would have to tailor this if statement to catch only this specific exception and not ignore it when actually something went wrong with authentication

Reply:
Maybe something using `PERMISSION_DENIED` or `USER_PERMISSION_DENIED` from the error response body above then? I don't know how stable that would be though.

Reply:
I commited a suggested method: https://github.com/Matatika/tap-googleads/pull/45#discussion_r1763071351